docsrc/conf.py

```diff
@@ -246,17 +237,21 @@
 
 # Generate the API documentation when building
 autosummary_generate = True
-numpydoc_show_class_members = True
-class_members_toctree = True
-numpydoc_show_inherited_class_members = True
+
+# avoid showing members twice
+numpydoc_show_class_members = False
 numpydoc_use_plots = True
 
+# automatically document class members
+autodoc_default_options = {"members": True}
+
 # display the source code for Plot directive
 plot_include_source = True
 
+
 def setup(app):
     app.add_stylesheet("pysal-styles.css")
 
-# Example configuration for intersphinx: refer to the Python standard library.
-intersphinx_mapping = {'https://docs.python.org/3.6/': None}
 
+# Example configuration for intersphinx: refer to the Python standard library.
+intersphinx_mapping = {"https://docs.python.org/3.6/": None}
```

Comment:
'numpy': ('https://docs.scipy.org/doc/numpy', None),
    'pandas': ('https://pandas.pydata.org/pandas-docs/stable/', None),
    'geopandas': ('https://geopandas.readthedocs.io/en/latest/', None),
    'sklearn': ('https://scikit-learn.org/stable/', None),
    'giddy': ('https://giddy.readthedocs.io/en/latest/', None),
    'libpysal': ('https://pysal.org/libpysal/', None),
    'esda': ('https://esda.readthedocs.io/en/latest/', None),
    'region': ('https://region.readthedocs.io/en/latest/', None)
}

Reply:
AH, yes! Will update that.

Reply:
maybe scipy too? not sure what you use in spaghetti

Reply:
doh, yeah. 

Also, should I have kept `'https://docs.python.org/3.6/': None` or is that OK to just get rid of?

Reply:
sorry, i meant those as addition not replacement. If you keep `'https://docs.python.org/3.6/': None` you get formatting for basic python primitives like str and int and dict